src/Block/Service/AbstractClassificationBlockService.php

```diff
@@ -56,6 +69,98 @@ public function __construct(
         }
     }
 
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/block-bundle 3.16, to be removed in version 4.0.
+     */
+    public function prePersist(BlockInterface $block)
+    {
+    }
+
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/block-bundle 3.16, to be removed in version 4.0.
+     */
+    public function postPersist(BlockInterface $block)
+    {
+    }
+
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/block-bundle 3.16, to be removed in version 4.0.
+     */
+    public function preUpdate(BlockInterface $block)
+    {
+    }
+
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/block-bundle 3.16, to be removed in version 4.0.
+     */
+    public function postUpdate(BlockInterface $block)
+    {
+    }
+
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/block-bundle 3.16, to be removed in version 4.0.
+     */
+    public function preRemove(BlockInterface $block)
+    {
+    }
+
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/block-bundle 3.16, to be removed in version 4.0.
+     */
+    public function postRemove(BlockInterface $block)
+    {
+    }
+
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/block-bundle 3.16, to be removed in version 4.0.
+     */
+    public function buildCreateForm(FormMapper $formMapper, BlockInterface $block)
+    {
+        $this->buildEditForm($formMapper, $block);
+    }
+
+    /**
+     * NEXT_MAJOR: Remove this method.
+     *
+     * @deprecated since sonata-project/block-bundle 3.16, to be removed in version 4.0.
+     */
+    public function buildEditForm(FormMapper $form, BlockInterface $block)
+    {
```

Comment:
```suggestion
    {
         $this->buildEditForm($formMapper, $block);
```

Reply:
This is AbstractAdminBlockService method, we should not change then. Self reference method make infinity loop.